data/json/monsters/drones.json

```diff
@@ -185,5 +185,19 @@
       "PRIORITIZE_TARGETS",
       "HIT_AND_RUN"
     ]
+  },
+  {
+    "id": "mon_tazer_hack",
+    "copy-from": "base_drone",
+    "type": "MONSTER",
+    "name": { "str": "tazer hack" },
+    "description": "An automated drone, this small quadcopter robot appears to have a tazer strapped to the front.  While extrememly fast, it is very fragile.",
```

Comment:
So it's makeshift strapped, not mass produced or manufactured as a complete product?  If makeshift who is making them and why?  I could absolutely see them as a product used in labs to subdue experiment subjects without killing them.  But then they'd be 3d printed to hold a tazer or some kind of similar design.

Reply:
Hm, yeah. That's a point of contention. Maybe "attached"